classes/local/manager.php

```diff
@@ -306,17 +306,18 @@ public static function check_file_storage_filesystem() {
     public static function get_available_fs_list() {
         $result[''] = get_string('pleaseselect', OBJECTFS_PLUGIN_NAME);
 
-        $filesystems['\tool_objectfs\azure_file_system'] = '\tool_objectfs\azure_file_system';
+        $filesystems['\tool_objectfs\azure_file_system'] = '\tool_objectfs\azure_file_system [DEPRECATED]';
```

Comment:
Just confirming that the value is only used for display, and isn't used as a key later on in selection?

Reply:
Yes that's what the change on 315 and 320 were for :+1: